.github/workflows/ci.yml

```diff
@@ -68,7 +48,7 @@ jobs:
         run: |
           bazelisk --bazelrc=ci.bazelrc test ${{ matrix.lib }}
           bazelisk --bazelrc=ci.bazelrc build ${{ matrix.lib }}
-          bazelisk --bazelrc=ci.bazelrc build ${{ matrix.lib }} --config analyser
+          bazelisk --bazelrc=ci.bazelrc --config analyser build ${{ matrix.lib }}
```

Comment:
Pretty sure this should be `bazelisk --bazelrc=ci.bazelrc build --config analyser ${{ matrix.lib }}`

Reply:
Argh, that's right!